src/components/task-form/task-form.component.js

```diff
@@ -0,0 +1,60 @@
+import { useRef, useState } from "react";
+import GetPriorityOptions from "../../services/get-priority-options";
```

Comment:
Similar to my [comment in the service file](https://github.com/Joao-Matias/to-do-list/pull/2/files#diff-5205f88e4c7d2b7f381a095fce630e162e2e10a9aa922f0dd8acaae5984ef0a0R9), it's best practice to have this named in snakeCase

Reply:
fce1599fba861da3c0a7cff34454d7be00ba2a38 Changed as requested.